Application/Frontend/pages/api/ID-to-User-Name.ts

```diff
@@ -0,0 +1,61 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { MongoClient } from 'mongodb';
+import { getMongoDbUri } from '@/lib/dbConfig';
+
+/**
+ * Handles the POST request for user registration, receiving the user's username, and find ONE entry that contains its username
```

Comment:
These docstrings are incorrect and will need to be updated. Looks like you took them from the `registration.ts` API endpoint. You will need to re-write these to reflect the docstrings for this endpoint.